docs/network/ble/ble_hs/ble_gatts/functions/ble_gatts_svc_set_visibility.md

```diff
@@ -0,0 +1,24 @@
+## <font color="#F2853F" style="font-size:24pt">ble\_gatts\_svc\_set\_visibility</font>
+
+```c
+int
+ble_gatts_svc_set_visibility(uint16_t handle, int visible)
+```
+
+### Description
+
+Changes visibility of a service with specified handle.
```

Comment:
Lets add here more description:
Maybe let's reuse Andrzejs commit msg for this description?

    This function allow any service to be hidden (and then restored) from clients. 
    Note: From GATT point of view, the service is still registered and has the same handle range assigned
    and it is ATT server which hides the attributes from the client.